config.py

```diff
@@ -819,6 +819,22 @@ def _setup_statistics_roles(cls) -> None:
             statistics_roles if statistics_roles else DEFAULT_STATISTICS_ROLES
         )
 
+    @classmethod
+    def _setup_membership_tied_roles(cls) -> None:
+        raw_membership_tied_roles: str = os.getenv("MEMBERSHIP_TIED_ROLES", default="").strip()
+
+        if not raw_membership_tied_roles:
+            cls._settings["MEMBERSHIP_TIED_ROLES"] = frozenset()
+            return
+
+        membership_tied_roles: AbstractSet[str] = {
+            raw_membership_tied_role.strip()
+            for raw_membership_tied_role in raw_membership_tied_roles.split(",")
+            if raw_membership_tied_role.strip()
+        }
+
+        cls._settings["MEMBERSHIP_TIED_ROLES"] = membership_tied_roles or frozenset()
```

Comment:
```suggestion
        cls._settings["MEMBERSHIP_TIED_ROLES"] = membership_tied_roles if membership_tied_roles else frozenset()
```
Use explicit falsey checking rather than error prone 'or'ing.

Reply:
realised we don't need it anyway cos it'll just be an empty frozen set either way so no need for the check at all.